qiskit/transpiler/passes/optimization/optimize_1q_gates.py

```diff
@@ -36,7 +37,7 @@ def __init__(self, basis=None, eps=1e-15):
         Args:
             basis (list[str]): Basis gates to consider, e.g. `['u3', 'cx']`. For the effects
                 of this pass, the basis is the set intersection between the `basis` parameter and
-                the set `{'u1','u2','u3'}`.
+                the set `{'u1','u2','u3', 'u'}`.
```

Comment:
Likewise, can you add `p` as an alias for `u1`?

Reply:
Yeah, that's a good idea. Just added this in: https://github.com/Qiskit/qiskit-terra/pull/5429/commits/e37928cf5cbfe998adda48ccab6b462bad02d1d9